src/config/winston.config.ts

```diff
@@ -0,0 +1,32 @@
+import winston from 'winston';
+import rootPath from 'app-root-path';
+
+const LOGS_PATH = `${rootPath}/logs`;
+
+const logger = winston.createLogger({
+    level: process.env.NODE_ENV === 'development' ? 'debug' : 'error',
+    format: winston.format.combine(
+        winston.format.timestamp(),
+        winston.format.json(),
+    ),
+    transports: [
+        new winston.transports.File({ filename: `${LOGS_PATH}/error.log`, level: 'error' }),
+        new winston.transports.File({ filename: `${LOGS_PATH}/combined.log` })
+    ],
+    exceptionHandlers: [
+        new winston.transports.File({ filename: `${LOGS_PATH}/exceptions.log`, level: 'error' }),
+        new winston.transports.Console()
+    ],
+    rejectionHandlers: [
+        new winston.transports.File({ filename: `${LOGS_PATH}/rejections.log`, level: 'error' }),
+        new winston.transports.Console()
+    ]
+});
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Consider going through the app and refactoring any "prod vs dev" conditions to follow unified approach.

Reply:
My grep-find might have failed me, but I think I only saw one with this discrepancy. Good point about unified approach (Similar to the issue to if/else over early return haha)